demo/app.js

```diff
@@ -551,17 +551,17 @@ const Demo = React.createClass({
               ref='drawer'
               title='This is the drawer component'
             >
-              <p style={{ padding: 20, fontFamily: 'Helvetica, Arial, sans-serif' }}>
+              <div style={{ padding: 20, fontFamily: 'Helvetica, Arial, sans-serif' }}>
```

Comment:
CSS expects certain elements when traversing the DOM tree so that you can do things like `div -> header -> a` to style all links within any header inside a div.  Because of this, it freaks out if you nest elements inside another element that are not supported.  Divs inside of  P tags happen to be one of those.  Since Button is a div, you can't place it in a p tag.  The easiest solution was to just make the p tags divs in this case.